### PR TITLE
[semver:patch] Add missing name on pull command

### PR DESCRIPTION
**Pull request by guillaumelecerf**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Motivation, issues

The pull command is missing a name, resulting in a weird display on the CircleCI GUI.

### Description

This adds a name to the pull command

### CODE REVIEW

**Comment by KyleTryon** [comment]
Hey @guillaumelecerf ,

You are definitely right we should add a name to this command. However, I don't think it's necessary to add a parameter for this, we can just go ahead and name it "docker pull" I think. Would you mind amending the PR?

File: src/commands/pull.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 description: Pull one or more Docker images from a registry
 
 parameters:
+  step-name:
```
Comment (KyleTryon) [inline src/commands/pull.yml:4]: I think we can do without the parameter for step name.

**Review by KyleTryon — CHANGES_REQUESTED** [review]

File: src/commands/pull.yml (hunk 1 above)
Comment (guillaumelecerf) [reply src/commands/pull.yml]: Done.

**Comment by guillaumelecerf** [comment]
@KyleTryon : done

**Review by gmemstr — APPROVED** [review]

**Review by KyleTryon — APPROVED** [review]

**Pull request merged**